docs/medical-api/api-reference/fhir/create-patient-consolidated.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "Create Patient's Consolidated Data"
+description: "Create Patient's Consolidated Data with a FHIR Bundle."
+api: "POST /medical/v1/patient/{id}/consolidated"
+---
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Body
+
+<ParamField body="resourceType" type="string" required>
+  The resource needs to be "Bundle"
+</ParamField>
+
+<ParamField body="type" type="string" required>
+  The type needs to be "collection"
+</ParamField>
```

Comment:
1. Since these are fixed, can't we remove the parameters?
2. If we want to keep those, why we have an enum of accepted values for `type` if the only one we accept is `collection`?